### PR TITLE
zero voxels no more included when computing MI + new flag to compute normalized MI

### PR DESCRIPTION
**Pull request by jcohenadad**
fixed issue #1132

### CODE REVIEW

File: scripts/sct_maths.py [hunk 1]
```diff
@@ -515,16 +523,34 @@ def compute_similarity(data1, data2, fname_out='', metric='', verbose=1):
     :param metric: 'mi' for mutual information or 'corr' for pearson correlation coefficient
     :return: None
     '''
+    import matplotlib
```
Comment (peristeri) [inline scripts/sct_maths.py]: Should you use matplotlib if verbose > 1 ?

Comment (jcohenadad) [inline scripts/sct_maths.py]: i though we can slowly start outputing by default for QC

Comment (peristeri) [inline scripts/sct_maths.py]: It would be best practice to isolate the visualization to make integration to other tools easier.

Comment (jcohenadad) [inline scripts/sct_maths.py]: ok i'll change it-- can you please put a note to discuss that during next sct meeting-- thanks

**Pull request merged**